sdk/communication/azure-communication-callingserver/pom.xml

```diff
@@ -42,7 +42,7 @@
     <test.dir>src/test</test.dir>
     <jacoco.min.linecoverage>0.70</jacoco.min.linecoverage>
     <jacoco.min.branchcoverage>0.70</jacoco.min.branchcoverage>
-    <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
+    <jacoco.skip.coverage.check>false</jacoco.skip.coverage.check>
```

Comment:
This is where the build failed. Either disable for now (true) or lower the coverage until it is raised

Reply:
using @DisabledIfEnvironmentVariable to disable some tests from LIVE tests, but they will be part of PLAYBACK, so coverage should be up now.

Reply:
Will revisit when these disabled tests are ready to be part of LIVE tests